runelite-client/src/main/java/net/runelite/client/plugins/ammo/AmmoPlugin.java

```diff
@@ -90,16 +95,26 @@ public void onItemContainerChanged(ItemContainerChanged event)
 			return;
 		}
 
-		checkInventory(event.getItemContainer().getItems());
+		checkInventory(event.getItemContainer());
+	}
+
+	@Subscribe
+	public void onGameTick(GameTick event)
+	{
+		checkQuiver();

```

Comment:
Not that it's particularly costly to do this every tick, but is there a reason this doesn't happen on var change?